localisation/00_memewc_united_localisation.csv

```diff
@@ -1,111 +1,111 @@
-ring_5_diplomacy_desc;Legendary Ring.;;Legend�rer Ring.;;;;;;;;;;;x
-ring_5_martial;[GetArtifactJewelryLegendaryMartialAffixN1] Ring[GetArtifactJewelryLegendaryMartialAffixN2];;[GetArtifactJewelryLegendaryMartialAffixN1] Ring[GetArtifactJewelryLegendaryMartialAffixN2];;;;;;;;;;;x
-ring_5_martial_desc;Legendary Ring.;;Legend�rer Ring.;;;;;;;;;;;x
-ring_5_stewardship;[GetArtifactJewelryLegendaryStewardshipAffixN1] Ring[GetArtifactJewelryLegendaryStewardshipAffixN2];;[GetArtifactJewelryLegendaryStewardshipAffixN1] Ring[GetArtifactJewelryLegendaryStewardshipAffixN2];;;;;;;;;;;x
-ring_5_stewardship_desc;Legendary Ring.;;Legend�rer Ring.;;;;;;;;;;;x
-ring_5_intrigue;[GetArtifactJewelryLegendaryIntrigueAffixN1] Ring[GetArtifactJewelryLegendaryIntrigueAffixN2];;[GetArtifactJewelryLegendaryIntrigueAffixN1] Ring[GetArtifactJewelryLegendaryIntrigueAffixN2];;;;;;;;;;;x
-ring_5_intrigue_desc;Legendary Ring.;;Legend�rer Ring.;;;;;;;;;;;x
-ring_5_learning;[GetArtifactJewelryLegendaryLearningAffixN1] Ring[GetArtifactJewelryLegendaryLearningAffixN2];;[GetArtifactJewelryLegendaryLearningAffixN1] Ring[GetArtifactJewelryLegendaryLearningAffixN2];;;;;;;;;;;x
-ring_5_learning_desc;Legendary Ring.;;Legend�rer Ring.;;;;;;;;;;;x
+RULE_ARTIFACT_SPAWN;�RWarCraft�!: Generic Artifacts Spawn At Gamestart ;�RWarCraft�! : Artefacts g�n�riques au d�but du jeu;�RWarCraft�!: Zuf�llige Artefakte bei Spielbeginn;;;;;;;;;;;x
+RULE_ARTIFACT_SPAWN_DEFAULT;Default;Par d�faut;Standard;;;;;;;;;;;x
+RULE_ARTIFACT_SPAWN_DEFAULT_DESC;Default amount of spawned artifacts on gamestart.;Quantit� par d�faut d'artefacts g�n�r�s lors du d�marrage du jeu.;Standardm��ige Anzahl der verteilten Artefakte beim Spielstart.;;;;;;;;;;;x
+RULE_ARTIFACT_SPAWN_X1_5_DESC;Amount of generic artifacts spawned on gamestart is modified by �G1.5�!.;La quantit� d'artefacts g�n�r�s lors du d�marrage du jeu est modifi�e par �G1.5�!.;Die Anzahl der zuf�lligen Artefakte, die beim Spielstart verteilt werden ist um �G1,5�! erh�ht.;;;;;;;;;;;x
+RULE_ARTIFACT_SPAWN_X2_DESC;Amount of generic artifacts spawned on gamestart is modified by �G2�!.;La quantit� d'artefacts g�n�r�s lors du d�marrage du jeu est modifi�e par �G2�!.;Die Anzahl der zuf�lligen Artefakte, die beim Spielstart verteilt werden ist um �G2�! erh�ht.;;;;;;;;;;;x
+RULE_ARTIFACT_SPAWN_X0_5_DESC;Amount of generic artifacts spawned on gamestart is modified by �R0.5�!.;La quantit� d'artefacts g�n�r�s lors du d�marrage du jeu est modifi�e par �G0.5�!.;Die Anzahl der zuf�lligen Artefakte, die beim Spielstart verteilt werden ist um �R0,5�! reduziert.;;;;;;;;;;;x
+RULE_ARTIFACT_SPAWN_X0_25_DESC;Amount of generic artifacts spawned on gamestart is modified by �R0.25�!.;La quantit� d'artefacts g�n�r�s lors du d�marrage du jeu est modifi�e par �G0.25�!.;Die Anzahl der zuf�lligen Artefakte, die beim Spielstart verteilt werden ist um �R0,25�! reduziert.;;;;;;;;;;;x
```

Comment:
```suggestion
RULE_ARTIFACT_SPAWN_X0_5_DESC;Amount of generic artifacts spawned on gamestart is modified by �R0.5�!.;La quantit� d'artefacts g�n�r�s lors du d�marrage du jeu est modifi�e par �R0.5�!.;Die Anzahl der zuf�lligen Artefakte, die beim Spielstart verteilt werden ist um �R0,5�! reduziert.;;;;;;;;;;;x
RULE_ARTIFACT_SPAWN_X0_25_DESC;Amount of generic artifacts spawned on gamestart is modified by �R0.25�!.;La quantit� d'artefacts g�n�r�s lors du d�marrage du jeu est modifi�e par �R0.25�!.;Die Anzahl der zuf�lligen Artefakte, die beim Spielstart verteilt werden ist um �R0,25�! reduziert.;;;;;;;;;;;x
```

Reply:
Done in 60f0f17513eb8ea4e52b163acd0cf6c28eeae598.